### ISSUE TITLE
Inner classes are missing

### ISSUE THREAD

**Issue by supersache** [ecd-plugin/ecd#94]
It is a bit of a mystery that noone has brought this up before... Regardless of which decompiler you chose, inner classes are always missing in the decompiled eclipse view. So are references to those, e.g. class attributes whose type is an inner class of another type.

**Comment by JanecekPetr** [comment]
I stumbled on ths issue because I am indeed seeing similar things. However, there are at least two different issues at hand, so let's make it more specific.

1. I have a jar (which I cannot distribute, but will try with a different one and report back) on which I call the "Export Source" action. Doesn't matter whether it's on the whole jar or a single package. This generates a zip file containing all decompiled .java sources ... **but without the inner classes**. If I specifically call "Export sources" on a single inner class, it works. If I select multiple class files, the inner classes are not included.

2. By default inner class files are not visible in the Project Explorer. This ties into point 1, actually point 1 makes this problem seem much worse than it is. It _looks_ like the decompiler does not see the inner class files at all because they're not in the Package explorer and they're not exported when decompiling the whole jar. However, this can be configured.
    In other words, the inner class files are hidden by default, but can be displayed at will, and decompiled one by one.

![obrazek](https://github.com/ecd-plugin/ecd/assets/1746404/d733f19b-0c8b-48ad-a418-8b04e72e439b)

What I want, obviously, is to fix the first issue. If that works as expected, the second issue would never even seem like an issue. Extra points for decompiling the inner classes automatically when only the outer class is selected for the extraction.
